src/main/java/nl/_42/max_login_attempts_spring_boot_starter/service/LoginAttemptServiceImplementation.java

```diff
@@ -87,8 +82,8 @@ public synchronized boolean loginFailed(String username, String remoteAddress) {
 
     private boolean hasReachedLoginAttemptLimit(String username, String remoteAddress) {
         UsernameIPAddress usernameIPAddress = new UsernameIPAddress(username, remoteAddress);
-        int attempts = attemptsCache.getOrDefault(usernameIPAddress, 0);
-        return attempts >= loginAttemptConfiguration.getMaxAttempts();
+        Attempt attempt = userCache.getOrDefault(usernameIPAddress, new Attempt());
```

Comment:
Is het nodig om deze opnieuw uit de cache te halen? Dat doe je in regel 62 toch al?

Reply:
Bij deze is dat niet nodig. Ik zal dit aanpassen.